src/content/docs/dropins/product-details/pdp-initialization.mdx

```diff
@@ -49,13 +49,11 @@ initializers.register(productApi.initialize, {
 
 ## Set default product options
 
-When a user navigates to the product detail page (PDP) on your site, you can set certain options as preselected defaults. This use case allows merchandisers to set default options through the Adobe Commerce Admin, which provides a more customized and streamlined shopping experience.
+When a user navigates to the product detail page (PDP) on your site, you can set certain options as preselected defaults for complex products. This use case allows merchandisers to set default options through the Adobe Commerce Admin, which provides a more customized and streamlined shopping experience.
 
-* **Simple products**—Default option selection is supported out-of-the-box through the Admin (`isDefault`). No additional PDP dropin customization is required.
+Default option selection is _not_ supported out-of-the-box for complex products. Instead, you must use [product attributes](https://experienceleague.adobe.com/en/docs/commerce-admin/catalog/product-attributes/product-attributes) and customize the PDP dropin initializers to define which attribute and value is used by default.
 
-* **Complex products**—Default option selection is _not_ supported out-of-the-box. Instead, you must use [product attributes](https://experienceleague.adobe.com/en/docs/commerce-admin/catalog/product-attributes/product-attributes) and customize the PDP dropin initializers to define which attribute and value is used by default. If no option is defined as default, the first listed option will be preselected by default.
-
-This approach allows merchants to set different default selections for complex products in the Admin. Developers can then customize the PDP dropin to show the default or preselected options.
+If no option is defined as default in the Admin for the dropdown input type, the first listed option will be preselected by default. You can opt out of the preselection in the dropdown by providing a placeholder (for example, "Choose an option").
```

Comment:
Can we leave out this part?
> You can opt out of the preselection in the dropdown by providing a placeholder (for example, "Choose an option").

This is conditioned by a flag that I've recently introduced, but I'd rather keep this separate for now.

Reply:
No problem. I removed it.